temporal/t.rast.neighbors/t.rast.neighbors.py

```diff
@@ -131,33 +194,63 @@ def main():
     nprocs = options["nprocs"]
     time_suffix = options["suffix"]
     new_labels = options["semantic_labels"]
+    quantiles = (
+        [float(quant) for quant in options["quantile"].split(",")]
+        if options["quantile"]
+        else None
+    )
+
+    if flags["r"] and options["region_relation"]:
+        gs.warning(_("Using current computational region for map selection."))
```

Comment:
r is to ignore the current region and region_relation is to process only maps with x spatial relation to the current computational region, shouldn't they be exclusive to avoid this warning? The warning also sounds confusing because using the current region is generally the default behavior, no?

Reply:
This is in princile implemented similar to `t.rast.univar`. So the user should be able to select maps by computational region but run computation on the entire maps.

The warning was intended to make the user aware that the region is used for selection but disregarded in processing... I agree that it may be confusing.

Do you have a suggestion for a better message (maybe change it to verbose?) or would it be better to drop the message alltogether, as `t.rast.univar` does not give a message like that...

Reply:
Now I understand. I'd change it to verbose or directly remove. In any case, if not yet there, maybe a note on the manual page explaining the behavior is worth it? i.e., that -r only affects maps selection but then those maps will be processed in full extent.

Reply:
Both suggestions are now implemented...